syntax = "proto3";

package modemCommunication;

service ModemCommunicationService{
  rpc ModemCommunication (ModemCommunicationRequest) returns (ModemCommunicationReply);
}

message ModemCommunicationRequest {
  string message = 1;
  int32 contact_number = 2;
}
message ModemCommunicationReply {
  string message = 1;
}
